src/components/sections/Team.section.tsx

```diff
@@ -0,0 +1,223 @@
+interface Member {
+    name: string;
+    title: string;
+    profile: string;
+}
+
+interface MemberProps extends Member {}
+
```

Comment:
I don't know how to feel about this.

I feel like the many components that the section is broken up to makes it hard to keep things clean.

Reply:
So you are saying, why we got Member and Department, instead of just Department?

Reply:
Not exactly, I feel like it makes it really easy to read but at the same time there are the interface names lol. Its not a deal breaker because it doesn't change anything. I am just nit-picking here lol.

Reply:
I feel it dude QQ